config/locales/hyrax.en.yml

```diff
@@ -274,7 +274,8 @@ en:
         profile: Profile
```

Comment:
once we are done adding translations to the en file - we should generate the other files

Reply:
there is an automated generation process for them.